src/main/scala/com/fulcrumgenomics/util/NumericCounter.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016 Fulcrum Genomics LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+package com.fulcrumgenomics.util
+
+import dagr.commons.CommonsDef._
+
+/**
+  * Super-simple class for counting occurrences of any [Numeric]. Will return
+  * zero for any item that has not been counted yet.  Implements some useful methods
+  * to compute statistics when the objects being counted are numeric types.
+  */
+class NumericCounter[T](implicit n: Numeric[T]) extends SimpleCounter[T] {
+  import n._
+
+  /** Returns the mean as a [Double]. */
+  def mean(): Double = {
+    var len = 0L
+    val sum = iterator.map { case (k, v) => len += v; k.toDouble() * v }.sum
+    sum / len
+  }
+
+  /** Returns the standard deviation as a [Double]. */
+  def standardDeviation(): Double = {
```

Comment:
Two things:
1) Could we call this `stddev`?
2) How about adding a default parameter so it's `stddev(mean: Double = mean())` so that it works as it does now, but if the caller wants both they're not paying to generate the mean twice?